EmeraldIOS/Utils/Alert/EmeraldAlert.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  Toash.swift
+//  EmeraldIOS
+//
+//  Created by Gustavo Forero on 6/19/19.
+//  Copyright © 2019 Condor Labs. All rights reserved.
+//
+
+import UIKit
+public protocol StatusType {
+    var backgroundColor: UIColor? { get }
+    var image: UIImage? { get }
+    var borderColor: UIColor? { get }
+}
+
+public enum ToastStatus: StatusType {
+    case success
+    case failure
+}
+
+public enum Duration: Double {
```

Comment:
```suggestion
public enum EmeraldAlertDuration: Double {
```
`Duration` it's very general